extensions/ext/GLSL_EXT_ray_flags_primitive_culling.txt

```diff
@@ -73,6 +74,40 @@ Modifications to the OpenGL Shading Language Specification, Version 4.60
       #define GL_EXT_ray_flags_primtive_culling          1
 
 
+Additions to Chapter 4 of the OpenGL Shading Language Specification
+(Variables and Types)
+
+    Modify Section 4.4, Layout Qualifiers
```

Comment:
Should probably add something to the SPIR-V mapping section?

I think the primitive_culling layout maps to the RayTraversalPrimitiveCullingProvisionalKHR capability.

Reply:
Updated.